test/utils/fixture/fixture.go

```diff
@@ -62,6 +62,16 @@ const (
 	OptionRandomizeName = 1 << iota
 )
 
+var (
+	teardownTime time.Duration
+	cooldownTime time.Duration
+)
+
+func init() {
+	cooldownTime = getCooldownTimeFromEnvVar()
+	teardownTime = getTeardownTimeFromEnvVar()
+}
+
```

Comment:
this is a good change because these 2 values are expected to be stable across the serial test suite run, so it makes sense (it's actually more correct) to fetch the value once at suite start